### PR TITLE
chore(release): 1.1.1

### PR DESCRIPTION
**Pull request by Francois-Esquire**

### CODE REVIEW

File: packages/holocron-module-register-webpack-plugin/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)",
     "Nickolas Oliver <nickolas.oliver@aexp.com> (https://github.com/PixnBits)"
   ],
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (JamesSingleton) [inline packages/holocron-module-register-webpack-plugin/package.json:20]: Just a comment, but why isn't the version at the top?

**Review by JamesSingleton — APPROVED** [review]

File: packages/holocron-module-register-webpack-plugin/package.json (hunk 1 above)
Comment (Francois-Esquire) [reply packages/holocron-module-register-webpack-plugin/package.json:20]: Hmmm I'm not sure..

Comment (JamesSingleton) [reply packages/holocron-module-register-webpack-plugin/package.json:20]: No need to change it right now, just thought it was odd

Comment (Francois-Esquire) [reply packages/holocron-module-register-webpack-plugin/package.json:20]: We can adjust it tho

**Review by JamesSingleton — APPROVED** [review]

**Review by mtomcal — APPROVED** [review]

**Review by JAdshead — APPROVED** [review]

**Pull request merged**